feature/soptamp/src/main/java/org/sopt/official/stamp/feature/ranking/rank/RankingViewModel.kt

```diff
@@ -109,10 +110,8 @@ class RankingViewModel @Inject constructor(
         SERVER("서버");
 
         companion object {
-            private val partMap = entries.associateBy { it.part }
-
-            fun getPartName(part: String): String? {
-                return partMap[part]?.name
+            fun getPartName(part: String): String {
+                return entries.find { it.part == part }?.name ?: throw throw IllegalArgumentException("Wrong Part Name")
```

Comment:
```suggestion
return entries.find { it.part == part }?.name ?: throw throw IllegalArgumentException("Wrong Part Name $part")
```

에러를 발생시킨 원인도 같이 에러로그에 남기면 좋겠죠?